Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM ubuntu:14.04
+FROM ubuntu:xenial
 MAINTAINER PyLabs pylabs@allegrogroup.com
 
 # set UTF-8 locale
+RUN apt-get clean && apt-get update
```

Comment:
Damn, I was living my life for a few days being sure I've commented here. It comes, I missed to press one extra button to do that.

I have moved this from the provisioner to the Dockerfile because locale-gen is used in here but it's not installed by default in xenial.